lib/call-cabal-project-to-nix.nix

```diff
@@ -149,11 +150,18 @@ let
       }
       else replaceSoureRepos rawCabalProject;
 
-  plan-nix = runCommand "plan-to-nix-pkgs" {
+  plan-nix = runCommand "plan-to-nix-pkgs" ({
     nativeBuildInputs = [ nix-tools ghc hpack cabal-install pkgs.rsync pkgs.git ];
     # Needed or stack-to-nix will die on unicode inputs
     LANG = "en_US.UTF-8";
-  } (''
+  } // (if plan-sha256 == null
+    then {}
+    else {
+      outputHashMode = "recursive";
+      outputHashAlgo = "sha256";
+      outputHash = plan-sha256;
+    })
```

Comment:
This looks like it's a different change about making plans optionally fixed-output?

Reply:
Yes. I'll move this to its own PR.